app/mailers/works_mailer.rb

```diff
@@ -15,6 +15,15 @@ def deposited_email
     mail(to: @user.email, subject: "Your deposit, #{@work.title}, is published in the SDR")
   end
 
+  def first_draft_reminder_email
+    @work = params[:work]
+    @user = @work.depositor
+
+    subject = "Reminder: New version of a deposit to the #{@work.collection.name} collection in the SDR is in progress"
```

Comment:
can you use `@work.collection_name` here so that this object doesn't need to depend on a deep graph of dependencies?

Reply:
will do...